client/webpack.config.dev.js

```diff
@@ -11,57 +11,81 @@ module.exports = {
   entry: './index.js',
   target: 'web',
   output: {
-  path: path.join(__dirname, '/dist/'),
-  filename: 'bundle.js',
-  publicPath: '/',
-},
+    path: path.join(__dirname, '/dist/'),
+    filename: 'bundle.js',
+    publicPath: '/'
+  },
   plugins: [
     new HtmlWebpackPlugin({
       template: './public/index.html',
       filename: './index.html'
     }),
     new webpack.DefinePlugin({
-      'process': {
-        'env': {
-          'G_SUITE_DOMAIN': JSON.stringify(process.env.G_SUITE_DOMAIN),
-          'CLIENT_ID': JSON.stringify(process.env.CLIENT_ID)
+      process: {
+        env: {
+          G_SUITE_DOMAIN: JSON.stringify(process.env.G_SUITE_DOMAIN),
+          CLIENT_ID: JSON.stringify(process.env.CLIENT_ID),
+          API_URI: JSON.stringify(process.env.REACT_APP_API_URI)
         }
       }
     }),
-    new BundleTracker({filename: './webpack-stats.json'}),
+    new BundleTracker({
+      filename: './webpack-stats.json'
+    }),
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.optimize.UglifyJsPlugin(),
     new webpack.LoaderOptionsPlugin({
       debug: true
     }),
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.EnvironmentPlugin({ NODE_ENV: 'development' }),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: 'development'
+    })
   ],
   module: {
     loaders: [
-      { test: /\.(js|jsx)$/, exclude: /node_modules/, loader: 'babel-loader', query: { presets: ['es2015', 'react'] } },
-      { test: /(\.css|scss)$/, loaders: ['style-loader', 'css-loader', 'sass-loader'] },
-      { test: /\.(jpeg|png|gif|svg|jpg)$/, loader: 'url-loader?limit=25000' },
-      { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file-loader' },
-      { test: /\.(woff|woff2)$/, loader: 'url?prefix=font/&limit=5000' },
-      { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=application/octet-stream' },
-      { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml' }
-    ],
+      {
```

Comment:
this has nothing to do with the changes or was there a reason you had to edit this? if not, you can revert this aspect back